assets/app/views/_overview-deployment.html

```diff
@@ -44,12 +44,14 @@ <h3 class="h4">
 
     <div class="component meta-data">
       created <relative-timestamp timestamp="rc.metadata.creationTimestamp"></relative-timestamp
-      ><span ng-if="rc.status.details.causes.length"
-        ><span>, triggered by 
-          <span class="deployment-trigger" ng-repeat="cause in rc.status.details.causes">
+      ><span ng-if="rc.causes.length"
+        ><span>, triggered by
+          <span class="deployment-trigger" ng-repeat="cause in rc.causes">
             <span ng-switch="cause.type">
-              <span ng-switch-when="ImageChange">a new image for {{cause.imageTrigger.from | imageObjectRef}}</span>
+              <!-- FIXME: Do I need a default namespace? -->
+              <span ng-switch-when="ImageChange">a new image for {{cause.imageTrigger.from | imageObjectRef : null : true}}</span>
```

Comment:
since we're asking the display to be shortened here, an empty fallback namespace is fine. you do need to handle ImageTrigger and/or from being empty, so `a new image<span ng-if="cause.imageTrigger.from"> for {{cause.imageTrigger.from | imageObjectRef : null : true}}</span>`